doc/src/site/sphinx/migration_guide.rst

```diff
@@ -9,6 +9,9 @@ From 3.32 to 3.34
 - On ``H2OConf``, the setters ``setClientIcedDir`` and ``setNodeIcedDir`` are replaced by ``setIcedDir`` and
   getters ``clientIcedDir`` and ``nodeIcedDir`` are replaced by ``icedDir``. Also the spark options
   ``spark.ext.h2o.client.iced.dir`` and ``spark.ext.h2o.node.iced.dir`` are replaced by ``spark.ext.h2o.iced.dir``.
+  The property ``spark.ext.h2o.node.iced.dir`` and the setter method ``setNodeIcedDir`` has no effect in all `3.30.x.y-z` versions.
+  If a user needs to set a custom iced directory for executors, he/she can set the property ``spark.ext.h2o.node.extra`` to ``-ice_root dir``,
+  where ``dir`` is a user-specified directory.
```

Comment:
I am not sure why we need this here if you are already adding it to the section "From 3.28.1 to 3.30". To me, it seems redundant. It would be good to get @satai and @michalkurka's opinion.

I would change "he/she" to "they"

Reply:
I added a notion about the problem to the paragraph since it's already mentioned there, but will remove it, np. Thanks for the feedback!